pkg/operator/render.go

```diff
@@ -24,6 +24,7 @@ type renderConfig struct {
 	APIServerURL           string
 	Images                 *RenderConfigImages
 	KubeAPIServerServingCA string
+	Infra                  configv1.Infrastructure
```

Comment:
if Infra was added to the `ControllerConfigSpec` above why add it separately into the `renderConfig` as a field? Wouldn't it just be `ControllerConfig.Infra`?

Reply:
Good catch, it is not needed.

Reply:
thanks!